autograd/numpy/fft.py

```diff
@@ -1,4 +1,5 @@
 from __future__ import absolute_import
+import numpy as np
```

Comment:
Don't do this. Insted use the autograd numpy, which is already imported as `anp`. This will ensure that everything in your gradients is differentiable (hence you'll be able to take higher derivatives).